src/WebApi/Application/UseCases/QuestionSet/DeleteQuestionSet/DeleteQuestionSetUseCase.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Application.Repositories;
+using System.Threading.Tasks;
+
+namespace Application.UseCases.QuestionSet.DeleteQuestionSet
+{
+    public class DeleteQuestionSetUseCase : IDeleteQuestionSetUseCase
+    {
+
+        private IOutputPort _outputPort;
+
+        private readonly IQuestionSetRepository _questionSetRepository;
+
+        public DeleteQuestionSetUseCase(IQuestionSetRepository questionSetRepository)
+        {
+            _questionSetRepository = questionSetRepository;
+        }
+
+        public async Task Execute(DeleteQuestionSetInput input)
+        {
+            var questionSet = await _questionSetRepository.GetById(input.Id);
+
+            if (questionSet is null)
+            {
+                _outputPort.NotFound();
+                return;
+            }
+
+            /*
+            bool isQuestionsRemoved = await _questionSetRepository.RemoveQuestionsFromList(questionSet.Id, input.InterviewQuestionIds);
+
+            if (!isQuestionsRemoved)
+            {
+                _outputPort.Invalid();
+                return;
+            }
+            */
+
```

Comment:
I'd probably get rid of the commented out code, since the pipeline seems to be crying about it. 😅 

_nothing groundbreaking, though_

```suggestion
```

Reply:
I thought it is not pretty clean. It would be better to remove that code.